pyrgg/utils.py

```diff
@@ -0,0 +1,61 @@
+# -*- coding: utf-8 -*-
+"""Pyrgg utility module."""
+
+
+def get_min_weight(weight_dic):
+    """
+    Get minimum weight value.
+
+    :param edge_dic: edge dictionary
+    :type edge_dic: dict
+    :param weighted: weighted flag
+    :type weighted: bool
+    :return: minimum weight value
+    """
+    return min([abs(w) for weights in weight_dic.values() for w in weights])
+
+
+def get_max_weight(weight_dic):
+    """
+    Get maximum weight value.
+
+    :param edge_dic: edge dictionary
+    :type edge_dic: dict
+    :param weighted: weighted flag
+    :type weighted: bool
+    :return: maximum weight value
+    """
+    return max([abs(w) for weights in weight_dic.values() for w in weights])
+
+
+def is_signed(weight_dic):
```

Comment:
You may retain the `has_self_loop`, `is_signed`, and `is_multigraph` functions for potential future use, but remember to include `#pragma: no cover` at their first line.